Sanntidsappen/ViewControllers/Departure/DepartureViewController.swift

```diff
@@ -325,7 +323,22 @@ extension DepartureViewController: CLLocationManagerDelegate {
 // MARK: DepartureSearchResultViewControllerDelegate
 
 extension DepartureViewController: DepartureSearchResultViewControllerDelegate {
-
+    
+    func previewDepartureAtIndexPath(_ viewController: DepartureSearchResultViewController, at indexPath: IndexPath) -> DepartureDetailsViewController? {
+        let stop = searchResultController.stops[indexPath.item]
+        return self.delegate?.getDetailsViewController(forStop: stop)
+    }
+    
+    func commitPreviewedViewController(_ viewController: DepartureSearchResultViewController, viewControllerToCommit: DepartureDetailsViewController) {
+        
+        RecentStopSearchData.shared.saveSearchToCoreData(stop: viewControllerToCommit.stop) { stops in
```

Comment:
Should this be extracted into a method?

Reply:
Don't think that is necessary at this moment. Should probably be optimized in other ways tho.